vars/fedora-32.yml

```diff
@@ -14,5 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+_rpm_key:
+  - key: "https://dl.tvcdn.de/download/linux/signature/TeamViewer2017.asc"
+    fingerprint: "8CAE012EBFAC38B17A937CD8C5E224500C1289C0"
+    state: "present"
+
+_yum_repository:
+  - file: "teamviewer"
+    name: "teamviewer"
+    description: "Fedora 32 - x86_64 - Teamviewer"
+    baseurl: "http://linux.teamviewer.com/yum/stable/main/binary-$basearch/"
+    enabled: true
+    gpgcheck: true
+    repo_gpgcheck: true
+    gpgkey: "http://linux.teamviewer.com/pubkey/currentkey.asc"
```

Comment:
This line should sync with above `key: "https://dl.tvcdn.de/download/linux/signature/TeamViewer2017.asc"`, please double confirm the original generated `/etc/yum.repo.d/teamviewer` file generated by install their official RPM directly ;-)